atomics/T1566.001/T1566.001.yaml

```diff
@@ -56,3 +56,86 @@ atomic_tests:
     cleanup_command: |
       Remove-Item #{jse_path} -ErrorAction Ignore
     name: powershell
+---
+attack_technique: T1566.001
+display_name: "Phishing: Spearphishing Attachment"
+atomic_tests:
```

Comment:
Hello @onuratali Thanks for contributing. Can you remove these 4 lines ? Since we already have them in the top, its not needed here.